content/blog/tikv-on-juicefs.md

```diff
@@ -0,0 +1,559 @@
+---
+title: "Getting Started with JuiceFS Using TiKV"
+date: 2021-09-08
+author: Herald Yu
+tags: ['Community', 'Cloud Native', 'Storage']
+---
+
+{{< figure src="/img/blog/tikv-on-juicefs/tikv-on-juicefs-banner.png" caption="" number="" >}}
+
+As a cloud-native distributed storage system, [JuiceFS](https://juicefs.com/?hl=en) was designed into a plug-in structure at the beginning of its birth to ensure that new technologies can be continuously integrated into the JuiceFS ecosystem. According to your needs, you can flexibly choose two core components, the data storage engine and the metadata engine, according to their needs.
+
+The data storage engine is mainly object storage. It supports almost all public and private cloud object storage services, as well as KV storage, WebDAV, and local disks. The metadata engine supports databases such as Redis, MySQL, PostgreSQL, and SQLite.
+
+The newly released JuiceFS v0.16 officially supports TiKV, a distributed key-value database, which further meets the requirements for elastic scaling in high-performance, large-scale data storage.
+
+In this article, I’ll share with you how to use JuiceFS and select TiKV as a metadata engine.
+
+
+## TiKV, a highly scalable KV database
+
+TiKV is a distributed transactional key-value database featuring high scalability, low latency, and ease of use. It easily processes over a trillion rows of data in petabytes.
+
+TiKV supports unlimited horizontal scaling and provides a distributed transaction interface that meets ACID compliance. It also ensures data consistency and high availability of multiple replicas using the [Raft protocol](https://raft.github.io/raft.pdf).
+
+{{< figure src="/img/blog/tikv-on-juicefs/tikv-architecture.png" caption="" number="" >}}
+
+TiKV was developed by [PingCAP](https://pingcap.com/about) and is a graduated project of the Cloud Native Computing Foundation (CNCF).
+
+
+## Install TiKV
+
+PingCAP provides TiUP, a package manager, which can easily install TiKV on Linux or macOS.
+
+
+#### Install TiUP
+
+This command downloads and installs the appropriate TiUP version to your system.
+
+```shell
+
+curl --proto '=https' --tlsv1.2 -sSf [https://tiup-mirrors.pingcap.com/install.sh](https://tiup-mirrors.pingcap.com/install.sh) | sh
+
```

Comment:
```suggestion
curl --proto '=https' --tlsv1.2 -sSf [https://tiup-mirrors.pingcap.com/install.sh](https://tiup-mirrors.pingcap.com/install.sh) | sh
```
We need to remove the extra blank lines in all code block; otherwise it looks awkward👇 
![image](https://user-images.githubusercontent.com/59654584/132449793-4e5a643b-de1f-4760-a6cf-7751e3807b6c.png)